alpine/src/main/scala/io/delta/alpine/internal/util/TypeUtils.scala

```diff
@@ -1,53 +0,0 @@
-/*
```

Comment:
I remove this file. The only place we need to use is creating LogStore. We can call `Class.forName(className, true, Thread.currentThread().getContextClassLoader)` directly.

Reply:
I'll implement this in the LogStore PR